core/templates/dev/head/css/oppia.css

```diff
@@ -1195,11 +1195,34 @@ pre.oppia-pre-wrapped-text {
   border-radius: 4px;
 }
 
+.oppia-editable-section:hover pre {
+	background-color: inherit;
+}
+
+.oppia-editable-section:hover oppia-noninteractive-link,
```

Comment:
Yep, that is totally fine. I will close this one.

Before that, though, PTAL at my response to your comment on line 1221. I think that, if it works, it might remove the need for a lot of the individual-component twiddling.